Trustroots/trustroots
# Remove all IIFEs

Now we are in npm/webpack world we don't need these to give us scope

#### Proposed Changes

I wanted to get rid of the all the angular-era IIFE's (you know, `(function(){ /* code goes here */ })()` that wraps everything) as they are no longer needed we are in webpack (each file is it's own scope).

I wrote a jscodeshift script to do it (see https://gist.github.com/nicksellen/cbe11ad0ca9073ce762c02c897e67d1c), I was thinking we can just keep it as a standalone script (not in repo), make one BIG change PR to change everything, but then people can run the script in their branches to minimise big change.

Whaddya think?

#### Testing Instructions

* site should work as before!

To run the script:

```
npm install -g jscodeshift
curl https://gist.githubusercontent.com/nicksellen/cbe11ad0ca9073ce762c02c897e67d1c/raw/cf137f497d2d891d42c02cc6260e13f46f8f6d9f/transform.js > /tmp/transform.js
jscodeshift --no-babel -t /tmp/transform.js modules
```

Fixes #
